DataDog/integrations-core
# Add kube-dns integration

### What does this PR do?

Adds the kube-dns integration (grabs the metrics from prometheus using the new PrometheusCheck mother class recently merged.

### Motivation

I need to follow the dns metrics of my kubernetes platform, so instead of having to go to prometheus, I'd rather have everything in Datadog.

### Testing Guidelines

It's the standard basic techniques. The check and tests are really simple


@hkaj does this check fit what you had in mind for the usage of the PrometheusCheck mother class?

Thanks
